pyproject.toml

```diff
@@ -81,7 +82,7 @@ packaging = ">=19.2"
 Pillow = "9.2.0"
 poetry-core = "1.3.2"
 pre-commit = "2.13.0"
-psycopg2-binary = "2.8.6"
+psycopg2-binary = "2.9.6"
```

Comment:
Upgraded only to get the M1 macOS wheel for easy local install, it's not that Django's psycopg2 support requires a newer version ([docs](https://docs.djangoproject.com/en/4.2/ref/databases/#postgresql-notes)).